scripts/import_map/update.ts

```diff
@@ -0,0 +1,13 @@
+import { parse } from "https://deno.land/std/flags/mod.ts";
+
+const flags = parse(Deno.args, {
+  string: ["import-map", "sdk"],
+  default: {
+    "import-map": `${Deno.cwd()}/import_map.json`,
+    "sdk": "../deno-slack-api/src/",
```

Comment:
Since this isn't a shared script is there a reason that the default is `deno-slack-api`?

Reply:
Good catch 💯  that was definitely a typo originating from me trying to land the SDK and API PR simultaneously, might rethink my approach to avoid these issues